README.md

```diff
@@ -78,10 +83,15 @@ A template of the `licenses.json` file can be found in the repository at `src/ma
     java -jar lpvs-1.0.0.jar
     ```
 
+   When running the application you will also be able to use command line to input all the same values associated with github and license on the fly, like so:
+   ```bash
+   java -jar lpvs-1.0.0.jar -Dgithub.login=<`my-login`> -Dgithub.token=<`my-token`> etc.
+   ```
+
    Or alternatively build and run the Docker container with LPVS:
    ```bash
     docker build -t lpvs .
-    docker run -p 7896:7896 --name lpvs -e LPVS_GITHUB_TOKEN=<`github.token`> lpvs:latest
+    docker run -p 7896:7896 --name lpvs -e GITHUB_TOKEN=<`github.token`> lpvs:latest
```

Comment:
@v-kerimov  In this line we also can set all required variables, not only GITHUB_TOKEN

Reply:
Yes, but this is made clear earlier in README.md in the lines 75-76.

Btw, wouldn't it be better to modify all the GITHUB and LICENSE env vars by adding LPVS_ prefix to them, so that the they had fewer chances to intersect with an env var used by some other programme?

Reply:
Yes, agree to rename variables